pkg/healthcheck/health_controller.go

```diff
@@ -182,17 +181,72 @@ func (hc *HealthController) Run(healthChan <-chan *ControllerHeartbeat, stopCh <
 
 }
 
+//RunServer starts the HealthController's server
+func (hc *HealthController) RunServer(stopCh <-chan struct{}, wg *sync.WaitGroup) error {
```

Comment:
Is this suppose to replace `Run`?

Reply:
Yes. Actually I split Run into RunServer and RunCheck. Put the former before CacheSyncOrTimeout and the latter after CacheSyncOrTimeout.

Reply:
I see, should we delete `Run` then? I still see it in this file

Reply:
I have deleted it